### DISCUSSION TITLE
Whisper insanely slow for larger audios >=5min

### DISCUSSION THREAD

**Discussion by arnavmehta7** [openai/whisper#756] [Q&A]
Hi, so I tested out whisper large with a A100 GPU for a 5.5min audio. Although for <=1min audio it generally is a lot faster but took around 5-10minutes on this single 5.5min sample. I wonder if there is anyway to increase the speed.

Thanks

**Comment by ehobson** [comment]


**Reply by arnavmehta7** [reply]
@ehobson What? No. It is using GPU

**Comment by SuperBruceJia** [comment]
@arnavmehta7 Have you found any solution?
